staging/src/k8s.io/kms/internal/plugins/_mock/go.work.sum

```diff
@@ -1,5 +1,7 @@
 cloud.google.com/go/compute v1.21.0 h1:JNBsyXVoOoNJtTQcnEY5uYpZIbeCTYIeDe0Xh1bySMk=
 cloud.google.com/go/compute v1.21.0/go.mod h1:4tCnrn48xsqlwSAiLf1HXMQk8CONslYbdiEZc9FEIbM=
+cloud.google.com/go/compute v1.23.0 h1:tP41Zoavr8ptEqaW6j+LQOnyBBhO7OkOMAGrgLopTwY=
```

Comment:
Is it necessary to change this file?

Reply:
No, will remove this. It was added automatically by the IDE 😄